data-serving/data-service/src/server.ts

```diff
@@ -1,9 +1,22 @@
 import app from './index';
+import DbClient from './common/db-client';
+
 
 // Start Express server.
-const server = app.listen(app.get('port'), () => {
+const server = app.listen(app.get('port'), async () => {
     console.log('App running at http://localhost:%d.', app.get('port'));
```

Comment:
localhost:%d is bound to become stale once we start exporting that server outside, can we stick to "listening on port xxx" instead?

Reply:
yeah this is outside the scope of this change but happy to update in a followup for sure

Reply:
Sure fine to do in another change, thanks :)

Reply:
ah I see you were really talking about just changing the message here instead of using an env variable -- just doing in this PR since it's easy peasy. Thanks!